pkgs/development/libraries/mesa/default.nix

```diff
@@ -275,7 +276,7 @@ in stdenv.mkDerivation {
     python3Packages.mako
     python3Packages.ply
     jdupes
-    glslang
+    (lib.getBin glslang)
```

Comment:
Please add the rationale as a comment.

Reply:
It provides the same libraries, glslang just builds it again. In fact, we even use spirv-tools to build glslang.

Reply:
I just want a comment in the code that explains why we must use `getBin` here so that anyone not familiar reading this does not need to research these details in order to understand what's going on.

Reply:
Alright, I added that.